packages/lesswrong/components/review/ReviewVoteTableRow.tsx

```diff
@@ -41,30 +42,43 @@ const styles = (theme: ThemeType) => ({
   },
   expanded: {
     background: "#eee"
+  },
+  highlight: {
+    paddingTop: 16,
+    paddingRight: 16,
+  },
+  backIcon: {
+    position: "absolute",
+    left:-40,
+    color: theme.palette.grey[400],
+    cursor: "pointer"
   }
 });
 
 const ReviewVoteTableRow = (
-  { post, dispatch, dispatchQuadraticVote, useQuadratic, classes, expandedPostId, currentQualitativeVote, currentQuadraticVote }: {
+  { post, dispatch, dispatchQuadraticVote, useQuadratic, classes, expandedPostId, currentQualitativeVote, currentQuadraticVote, setExpandedPost }: {
     post: PostsList,
     dispatch: React.Dispatch<vote>,
     dispatchQuadraticVote: any,
+    setExpandedPost: any,
     useQuadratic: boolean,
     classes:ClassesType,
     expandedPostId: string,
     currentQualitativeVote: vote|null,
     currentQuadraticVote: quadraticVote|null,
   }
 ) => {
-  const { PostsTitle, LWTooltip, PostsPreviewTooltip, MetaInfo, QuadraticVotingButtons, ReviewVotingButtons } = Components
+  const { PostsTitle, LWTooltip, PostsPreviewTooltip, MetaInfo, QuadraticVotingButtons, ReviewVotingButtons, PostsHighlight } = Components
 
   const currentUser = useCurrentUser()
   if (!currentUser) return null;
+  const expanded = expandedPostId == post._id
```

Comment:
Never use `==`, basically always use `===` unless you are very intentional about it